src/external/wb_presets.c

```diff
@@ -2350,6 +2352,74 @@ const wb_data wb_preset[] =
   { "Nikon", "Coolpix P340", Cloudy, 0,			{ 1.976563, 1, 1.457031, 0 } },
   { "Nikon", "Coolpix P340", Flash, 0,			{ 2.085938, 1, 1.554688, 0 } },
 
+  // Nikon CoolPix P1000 firmware 1.2
+  { "Nikon", "COOLPIX P1000", HTMercury           , 0, { 1.76953125, 1, 1.61328125, 0 } },
+  { "Nikon", "COOLPIX P1000", DirectSunlight      , 0, { 1.4375, 1, 1.99609375, 0 } },
+  { "Nikon", "COOLPIX P1000", DirectSunlight      , 0, { 1.46875, 1, 1.94140625, 0 } },
+  { "Nikon", "COOLPIX P1000", DirectSunlight      , 0, { 1.50390625, 1, 1.88671875, 0 } },
+  { "Nikon", "COOLPIX P1000", DirectSunlight      , 0, { 1.5390625, 1, 1.8203125, 0 } },
+  { "Nikon", "COOLPIX P1000", DirectSunlight      , 0, { 1.578125, 1, 1.7578125, 0 } },
+  { "Nikon", "COOLPIX P1000", DirectSunlight      , 0, { 1.61328125, 1, 1.6953125, 0 } },
+  { "Nikon", "COOLPIX P1000", DirectSunlight      , 0, { 1.40625, 1, 2.05078125, 0 } },
+  { "Nikon", "COOLPIX P1000", Daylight            , 0, { 1.50390625, 1, 1.88671875, 0 } },
+  { "Nikon", "COOLPIX P1000", Shade               , 0, { 1.87890625, 1, 1.4140625, 0 } },
+  { "Nikon", "COOLPIX P1000", Cloudy              , 0, { 1.61328125, 1, 1.765625, 0 } },
+  { "Nikon", "COOLPIX P1000", Cloudy              , 0, { 1.65234375, 1, 1.71875, 0 } },
+  { "Nikon", "COOLPIX P1000", Cloudy              , 0, { 1.734375, 1, 1.6015625, 0 } },
+  { "Nikon", "COOLPIX P1000", Cloudy              , 0, { 1.7734375, 1, 1.546875, 0 } },
+  { "Nikon", "COOLPIX P1000", Cloudy              , 0, { 1.578125, 1, 1.8203125, 0 } },
+  { "Nikon", "COOLPIX P1000", Cloudy              , 0, { 1.54296875, 1, 1.87109375, 0 } },
+  { "Nikon", "COOLPIX P1000", Cloudy              , 0, { 1.69140625, 1, 1.66015625, 0 } },
+  { "Nikon", "COOLPIX P1000", Tungsten            , 0, { 1.015625, 1, 2.8046875, 0 } },
+  { "Nikon", "COOLPIX P1000", Incandescent        , 0, { 0.9921875, 1, 2.88671875, 0 } },
+  { "Nikon", "COOLPIX P1000", Incandescent        , 0, { 0.96875, 1, 2.96875, 0 } },
+  { "Nikon", "COOLPIX P1000", Incandescent        , 0, { 1.015625, 1, 2.8046875, 0 } },
+  { "Nikon", "COOLPIX P1000", Incandescent        , 0, { 1.0625, 1, 2.6171875, 0 } },
+  { "Nikon", "COOLPIX P1000", Incandescent        , 0, { 1.0390625, 1, 2.7109375, 0 } },
+  { "Nikon", "COOLPIX P1000", Incandescent        , 0, { 0.94921875, 1, 3.05078125, 0 } },
+  { "Nikon", "COOLPIX P1000", Incandescent        , 0, { 1.08984375, 1, 2.5234375, 0 } },
```

Comment:
This causes finetuning for `Incadescent` to be not taken into consideration. Again, all values for finetuning are `0`.

Also there are another weird coefficeint jums so we can't be sure in what order are those.